koach/Makefile

```diff
@@ -0,0 +1,9 @@
+CURDIR  := $(shell pwd)
+
+build:
+	cd $(CURDIR); go mod tidy
+	cd $(CURDIR); go build -o kubearmor-koach main.go
+	
+build-image:
+	cd $(CURDIR); docker build -t nathanieljason/koach-test:latest .
```

Comment:
Can you please use kubearmor docker hub repo to upload the image?
I have created [the secrets tokens](https://github.com/kubearmor/koach/settings/secrets/actions) in the koach repo.